src/commands/image/updateImage/UpdateImageDraftStep.ts

```diff
@@ -29,8 +29,7 @@ export class UpdateImageDraftStep<T extends UpdateImageContext> extends Revision
         this.revisionDraftTemplate.containers.push({
             env: context.environmentVariables,
             image: context.image,
-            // We need the revision draft to always show up as having unsaved changes, we can ensure this by adding a unique ID at end of the container name
-            name: getContainerNameForImage(nonNullProp(context, 'image')) + `-${randomUtils.getRandomHexString(5)}`,
+            name: getContainerNameForImage(nonNullProp(context, 'image')),
```

Comment:
Based on Anthony's comments, we probably don't want to be updating container apps with identical image names.  I put in the different name originally to still allow us to deploy the same image name, but now there's probably no need for this behavior.